server_plugin/server.py

```diff
@@ -189,6 +195,9 @@ def delete(vca_client, **kwargs):
 @operation
 @with_vca_client
 def get_state(vca_client, **kwargs):
```

Comment:
You also need to remove get_state(). If we leave it, _get_state() will be processed twice - in start() and in get_state()